eex/datalayer.py

```diff
@@ -1182,11 +1217,14 @@ def remove_terms(self, order, index=None, propogate=False):
         """
         order = metadata.sanitize_term_order_name(order)
 
+        # Check that terms with order exist
+
+
         # Initialize order list
         order_list = [order]
 
         # If this action should be propagated, orders will be added to order list.
-        if propogate == True:
+        if propagate == True:
```

Comment:
`True` is a singleton, therefore one should use `if propagate is True` generally.